pkg/daemon/ceph/osd/volume.go

```diff
@@ -220,6 +222,8 @@ func getCephVolumeOSDs(context *clusterd.Context, clusterName string, cephfsid s
 			if osd.Type == "journal" {
 				isFilestore = true
 			}
+			osdDevice = strings.Replace(osd.Devices[0], "/", "", 1)
+			osdDevice = strings.Replace(osdDevice, "/", "_", -1)
```

Comment:
Can't we just get `sdb` instead of `dev_sdb`, I don't see much value in having `dev_`.

Reply:
fine with me if there is a desire to move forward with this change.